sql/core/src/test/scala/org/apache/spark/sql/IntegratedUDFTestUtils.scala

```diff
@@ -200,8 +205,11 @@ object IntegratedUDFTestUtils extends SQLHelper {
       "PYSPARK_DRIVER_PYTHON", sys.env.getOrElse("PYSPARK_PYTHON", "python3"))
     if (TestUtils.testCommandAvailable(pythonExec)) {
       pythonExec
+    } else if (pythonExec.equalsIgnoreCase("python") &&
```

Comment:
@dongjoon-hyun, so just to clarify, can the changes in this file be removed because Jenkins uses `python` and GitHun Actions uses `python3` for now, and the tests pass correctly with the use of `command -v`?

Reply:
Sure!